kubeflow/trainer/rhai/instrumentation/checkpoint.py

```
@@ -0,0 +1,1014 @@
+                            uploaded_bytes += file_size
+                            if time.time() - last_update >= 1:
+                                mb = uploaded_bytes / (1024 * 1024)
+                                pct = int((uploaded_bytes / total_size) * 100) if total_size else 0
+                                _log(
+                                    f"Progress: {mb:.1f}"
+                                    f"/{total_size / (1024 * 1024):.1f} MB ({pct}%)"
+                                )
+                                last_update = time.time()
+
+                        return True
+                    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)